tests/src/unit/core/formatConversion/export/__snapshots__/html/hardbreak/multiple.html

```diff
@@ -1,4 +1,9 @@
-<p>
+<p
+  class="bn-block-content"
+  data-node-type="blockOuter"
```

Comment:
should this get added in this PR?

Reply:
Looks like you were reviewing only the changes from commit `13acc7e`, since that's what GitHub shows when I click the link to see the full diff. Has since been fixed👍